### PR TITLE
Update expected memory usage after attribute fixed cost has been redu…

### PR DESCRIPTION
**Pull request by baldersheim**
…ced.

@hmusum or @geirst or @toregge PR

### CODE REVIEW

**Review by hmusum — APPROVED** [review]

File: tests/search/metrics/metrics.rb [hunk 1]
```diff
@@ -24,7 +24,7 @@ def test_metrics
     assert_hitcount("f1:xyzzy", 0)
 
     metrics = vespa.search["test"].first.get_total_metrics
-    dump_metric_names(metrics)
+    # dump_metric_names(metrics)
```
Comment (geirst) [inline tests/search/metrics/metrics.rb:27]: Was this change intentional?

**Review by geirst — COMMENTED** [review]
👍

File: tests/search/metrics/metrics.rb (hunk 1 above)
Comment (baldersheim) [reply tests/search/metrics/metrics.rb:27]: Yes, I did not see its value. It just polluted the test output.
